src/components/Patient/PatientDetailsTab/Appointments.tsx

```diff
@@ -127,14 +127,16 @@ export const Appointments = (props: PatientProps) => {
                   </TableCell>
                   <TableCell>{getStatusBadge(appointment.status)}</TableCell>
                   <TableCell className="text-right">
```

Comment:
Since there are no actions present, it wouldn't make sense to display the column anymore, so you will want to make the heading and the cell conditional.